schema.ts

```diff
@@ -98,7 +107,7 @@ export const chats = pgTable(
     // of that chat.
     crudPolicy({
       role: authenticatedRole,
-      read: sql`(select auth.user_id() in (select user_id from chat_participants where chat_id = ${table.id}))`,
+      read: sql`((select auth.user_id()) = ${table.ownerId} or (select auth.user_id()) in (select user_id from MY_CHATS_PARTICIPANTS where chat_id = ${table.id}))`,
```

Comment:
We needed to add this `(select auth.user_id()) = ${table.ownerId} or` clause because `rls` rules use the existing `rls` rules to evaluate the data they can access, thus, without that first clause, we couldn't insert the first `chatParticipant` of each chat, because we couldn't access the `chats` table in the first place - the initial rule reading `chats` stated that we could only read the chats where we were chat participants.